src/Form/CivicrmEntitySettings.php

```diff
@@ -173,6 +174,8 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
     $this->routeBuilder->rebuild();
     $this->localActionManager->clearCachedDefinitions();
     $this->localTaskManager->clearCachedDefinitions();
+    // @todo inject views.views_data and call \Drupal\views\ViewsData::clear().
+    Cache::invalidateTags(['views_data']);
```

Comment:
I think we can delete this.

Reply:
Yeah, it's not needed. Entity definitions are always present, so the Views data always exists.